src/_posts/databases/mysql/2000-01-01-backing-up.md

```diff
@@ -0,0 +1,273 @@
+manually, whenever you want.
+
+### Retention Policy for Manual Backups
+
+We keep a limited amount of Manual backups depending on your database plan:
+
+<div class="overflow-horizontal-content" markdown="1">
+| Plan         | Backups Retained |
+| ------------ + ---------------- |
+| Sandbox      | N/A              |
+| Starter      | 10               |
+| Business     | 50               |
+{: .table }
+</div>
+
+When a database is removed from an application, the retention policy remains
+untouched: backups are **not** instantly deleted.
+
+### Creating a Manual Backup
+
+#### Using the Database Dashboard
+
+1. From your web browser, open your [database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
+2. Click the **Backups** tab
+3. Locate the **Backup Settings** block
+4. Click the **Make Manual Backup** button
+
+#### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the addons attached to your application:
+   ```bash
+   scalingo --app my-app addons
+   ```
+   The output should look like this:
+   ```text
+   +-------+-----------------------------------------+-------------------+---------+
+   | ADDON |                   ID                    |       PLAN        | STATUS  |
+   +-------+-----------------------------------------+-------------------+---------+
+   | MySQL | ad-871546ad-943a-4929-9770-ec7c839d65f5 | mysql-starter-512 | running |
+   ...
+   ```
+3. Locate the `ID` corresponding to the addon for which you want to configure
+   the backup schedule
+4. Ask the platform to backup the database:
+   ```bash
+   scalingo --app my-app --addon <addon_ID> backups-create
+   ```
+   After a while, the output should look like this:
+   ```text
+   -----> Backup successfully finished
+   ```
+
+### Downloading a Manual Backup
+
+Please refer to [Downloading an On-Demand Backup](#downloading-an-on-demand-backup)
+section, as the process is exactly the same.
+
+
+## Dumping the Database
+
+There are different ways to dump (and restore) a MySQL® database. We
+generally advise to either:
+- Process from a [One-Off container](#from-a-one-off-container)
+- Or conduct the operations [from your workstation](#from-your-workstation), by
+  accessing the database in a secured manner
+
+### From a One-Off Container
+
+This method has two main advantages:
+- It doesn't require to make your database reachable over Internet
+- You won't have to tweak your connection URI
+
+1. Follow the procedure to [access your MySQL® database from a One-Off
+   container]({% post_url databases/mysql/2000-01-01-accessing %}#using-a-one-off)
+2. From the One-Off command line, download the Scalingo Command Line Tool
+   using our `install-scalingo-cli` helper tool:
+   ```bash
+   install-scalingo-cli
+   ```
+3. Login to Scalingo:
+   ```bash
+   scalingo login
+   ```
```

Comment:
issue: I don't understand why you need the Scalingo CLI to backup the database. You don't use it in the next steps.

Reply:
Exact. I don't know what's happened here.
Fixed in https://github.com/Scalingo/documentation/pull/2501/commits/a09a0db9160463946bf57f30ad1092204161e7b9